src/components/Swap/Output.tsx

```diff
@@ -43,7 +42,7 @@ export default function Output() {
       <OutputWrapper
         isWide={isWideWidget}
         field={Field.OUTPUT}
-        impact={impact}
+        impact={fiatValueChange}
```

Comment:
we should show the fiat value change here, because right next to the USD value of the output.

Reply:
as a followup we should probably just rename the prop here